app/Http/Controllers/TalksController.php

```diff
@@ -102,7 +85,7 @@ public function store()
     public function edit($talkId)
     {
         try {
-            $talk = Auth::user()->talks()->findOrFail($talkId);
+            $talk = Auth::user()->talks()->withTrashed()->findOrFail($talkId);
```

Comment:
Hm, soft deletes and archive aren't usually the same thing. I get the idea of using soft deleted entries as archived talks, but that's not really how they're intended to be used, and I think this could lead to a lot of confusion. I'd prefer a "is_deleted" flag or status of some sort that we can then do a global scope on, or something similar.